docs/src/modules/java/pages/quickstart-template.adoc

```diff
@@ -294,14 +294,22 @@ To deploy your service to Kalix:
 
 [.group-java]
 First make sure you have updated the `dockerImage` property in the `pom.xml` to point at your Docker registry. Then:
+[.group-java]
+- Run `mvn deploy kalix:deploy`.
 
-- Run [.group-java]#`mvn deploy`,# [.group-scala]#`sbt Docker/publish`# which conveniently packages and publishes your Docker image prior to deployment.
+[.group-java]
+`deploy` packages and publishes your Docker image to your repository and `kalix:deploy` deploys the service with your image to Kalix. If `kalixProject` is set, that is the project where the service will be deployed. If it is not set, the service will be deployed in your currently selected project.
+[.group-java]
+IMPORTANT: If you time stamp your image. For example, `<dockerTag>${project.version}-${build.timestamp}</dockerTag>` you must always run both targets in one pass, i.e. `mvn deploy kalix:deploy`. You cannot run `mvn deploy` first and then `mvn kalix:deploy` because they have different timestamps and thus different `dockerTag`s. Making it impossible to reference the image in the repository from the second target.
+[.group-java]
+When using the CLI, in your command window, set your Kalix project to be the current project:
 
-- When using the CLI, in your command window, set your Kalix project to be the current project:
-+
 [source, command window]
 ----
 kalix config set project <project-name>
 ----
 
+[.group-scala]
+- Run `sbt Docker/publish` which conveniently packages and publishes your Docker image prior to deployment.
+
```

Comment:
Not quite sure, but, Isn't this doc page only for Java?

Reply:
There is Scala as well. One of them for example in https://github.com/lightbend/kalix-jvm-sdk/pull/1488/files/fb4ddf38f72425a263d6f7d0f5258dd29f316eb5#diff-7480dbc87d97e7210bc9502c753247865fc0994dc20ce5e0f03f9ca06a7543a4L74